### PR TITLE
updates to dev apps for broker

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: tests/devapps/XamarinDev/XamarinDev/App.xaml.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public static void InitPublicClient()
 
             if (UseBroker)
             {
-                //builder.WithBroker(true);
+                builder.WithBroker(true);
```
Comment (henrik-me) [inline tests/devapps/XamarinDev/XamarinDev/App.xaml.cs]: >true [](start = 35, length = 4)

 Can't this simply be "empty" not sure where this dicussion landed previously

File: tests/devapps/XamarinDev/XamarinDev.iOS/AppDelegate.cs [hunk 2]
```diff
@@ -33,7 +33,7 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
             Telemetry.GetInstance().RegisterReceiver(
                 (new Microsoft.Identity.Client.AriaTelemetryProvider.ClientTelemetryHandler()).OnEvents);
 #endif
-
+            App.RootViewController = new UIViewController();
             //App.MsalPublicClient.iOSKeychainSecurityGroup = "com.microsoft.adalcache";
```
Comment (henrik-me) [inline tests/devapps/XamarinDev/XamarinDev.iOS/AppDelegate.cs]: >//App. [](start = 12, length = 6)

Remove? Add comment?

File: tests/devapps/XamarinDev/XamarinDev/App.xaml.cs (hunk 1 above)
Comment (jennyf19) [reply tests/devapps/XamarinDev/XamarinDev/App.xaml.cs]: @henrik-me yes...you're right. this is copy/paste from before those changes...thanks for pointing it out.

**Comment by jennyf19** [comment]
[ui build passed here](https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=367882)

**Pull request merged**